reference/auth/models/coastalrequest.v1.yaml

```diff
@@ -0,0 +1,26 @@
+title: Coastal Request Data
+description: Data to send to Coastal's API.
```

Comment:
This could be improved with details. Which Coastal API? For what purpose? To what end?

If the API is published anywhere publicly, it might be nice to link to it.